src/styles/index.scss

```diff
@@ -1,3 +1,50 @@
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc($cell-size * 7 + $gap * 6);
```

Comment:
The variable `$cell-size` is used here, but it's not defined within this file. Ensure that `$cell-size` is defined before it's used, or import the file where it's defined.